source/javascripts/lib/wirsing/filters/search.coffee

```diff
@@ -0,0 +1,21 @@
+#= require ../module
+
+# Returns a configurable search function that can be used
+# to filter an array of objects by searching for a term
+# within the specified property of each object.
+
+wirsing.module 'filter.search', ->
+
+  (options) ->
+
+    search = (items) ->
+      if search.term.length < search.minChars then return items
+      else items.filter (item) -> item[search.in].search(search.term) > -1
```

Comment:
This function is gonna cause serious problems when `search.configure` hasn't been called before. Some default values that get set on initialization would solve that.